skimage/restoration/inpaint.py

```diff
@@ -126,25 +252,88 @@ def inpaint_biharmonic(image, mask, multichannel=False):
     image = skimage.img_as_float(image)
     mask = mask.astype(bool)
```

Comment:
```suggestion
    mask = mask.astype(bool, copy=False)
```
We may avoid a copy here :wink: